src/Utility/Security.php

```diff
@@ -48,16 +48,6 @@ class Security
     protected static $_instance;
 
     /**
-     * Generate authorization hash.
-     *
-     * @return string Hash
-     */
-    public static function generateAuthKey()
```

Comment:
Given the issues in #6139 we could nerf this method and throw exceptions. While this is also a breaking change, it is an easier to debug one than simply removing the method.